src/server/server.js

```diff
@@ -47,8 +48,15 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(requestIp.mw());
 
+const checkAuthorizationHeader = (req, res, next) => {
+  if (req.headers.authorization !== `ApiKey ${config.SERVER_API_KEY}`) {
```

Comment:
The `req.headers` object received in the middleware has the `authorization` key and not the `Authorization` key.